nixos/modules/services/monitoring/datadog-agent.nix

```diff
@@ -186,7 +186,7 @@ in {
       type = types.attrs;
       default = {
         init_config = {};
-        instances = [ { use-mount = "no"; } ];
+        instances = [ { use-mount = "false"; } ];
```

Comment:
What's this change for? Was the old value invalid?

Reply:
This parameter found in example config https://github.com/DataDog/datadog-agent/blob/master/cmd/agent/dist/conf.d/disk.d/conf.yaml.default#L6